index.js

```diff
@@ -263,6 +269,14 @@ function parse(input, options) {
 	}, Object.create(null));
 }
 
+function isEmptyString(value) {
+	return value === '';
+}
```

Comment:
This utility has no value over just checking directly.

Reply:
done